tests/unit/clearSession.test.ts

```
@@ -0,0 +1,46 @@
+import { mockClient } from "aws-sdk-client-mock";
+import init from "../../src/api/server.js";
+import { createJwt } from "./utils.js";
+import { secretObject } from "./secret.testdata.js";
+import supertest from "supertest";
+import { afterEach } from "node:test";
+import jwt from "jsonwebtoken";
+
+const app = await init();
+
+const ddbMock = mockClient(DynamoDBClient);
+const jwt_secret = secretObject["jwt_key"];
```

ESLint

[error] ["jwt_key"] is better written in dot notation.

(dot-notation)